app/github_public_repo/README.md

```diff
@@ -0,0 +1,20 @@
+
+# SARS-CoV-2 Serosurveillance Data by Serotracker
+
+SeroTracker synthesizes findings from thousands of COVID-19 seroprevalence studies worldwide, providing a data platform and interactive dashboard for SARS-CoV-2 serosurveillance. This dataset represents our collection of serosurveillance studies
```

Comment:
Let's add in the link to our website (maybe hyperlink interactive dashboard). Also change:

"The dataset in <insert name of csv for them to find> represents our collection of serosurveillance studies."